<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScatteredLettersStage</class>
 <widget class="QWidget" name="ScatteredLettersStage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="errorsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;p align=&quot;right&quot;&gt;&lt;font color=&quot;#0000ff&quot;&gt;&lt;b&gt;Errors: 0&lt;/b&gt;&lt;/font&gt;&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="translationLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QStarDict::ScatteredLetters" name="scatteredLetters" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="surrenderButton">
     <property name="text">
      <string>I cannot remember the word (Enter)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextWordButton">
     <property name="text">
      <string>Next word (Enter)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextStageButton">
     <property name="text">
      <string>Next stage (Enter)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QStarDict::ScatteredLetters</class>
   <extends>QWidget</extends>
   <header>scatteredletters.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
